plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -209,35 +206,23 @@ private FileWriter createParquetFileWriter(String path, List<DeltaLakeColumnHand
         try {
             Closeable rollbackAction = () -> fileSystem.deleteFile(path);
 
-            List<Type> parquetTypes = dataColumns.stream()
-                    .map(column -> {
-                        Type type = column.getType();
-                        if (type instanceof TimestampWithTimeZoneType timestamp) {
-                            verify(timestamp.getPrecision() == 3, "Unsupported type: %s", type);
-                            return TIMESTAMP_MILLIS;
-                        }
-                        return type;
-                    })
-                    .collect(toImmutableList());
-
-            List<String> dataColumnNames = dataColumns.stream()
-                    .map(DeltaLakeColumnHandle::getName)
+            List<ParquetTypeMapping> parquetTypeMappings = dataColumns.stream()
```

Comment:
i don't fully follow why `ParquetTypeMapping` is introduced.

We create it here, but we use it only in the `ParquetSchemaConverter`.
When passing values to the `ParquetFileWriter` we have to re-map the values to what the writer needs.

Reply:
outdated due to removed ParquetTypeMapping